src/style.css

```diff
@@ -1 +1,67 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.page {
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__search {
+  box-sizing: border-box;
+  outline: none;
+  width: 100%;
+  height: 100%;
+
+  background-repeat: no-repeat;
+  background-image: url(./images/Search.svg);
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  transition: box-shadow 0.3s ease;
+
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
You have used the 'Avenir' font, but there is no fallback font. If the 'Avenir' font fails to load for some reason, the text might not be displayed properly. It's recommended to always include a list of fallback fonts. For instance: 'font-family: Avenir, Arial, Helvetica, sans-serif;'